apis/externalsecrets/v1beta1/secretstore_gitlab_types.go

```diff
@@ -28,6 +28,9 @@ type GitlabProvider struct {
 
 	// ProjectID specifies a project where secrets are located.
 	ProjectID string `json:"projectID,omitempty"`
+
+	// Environment environment_scope of gitlab CI/CD variables
```

Comment:
Could you please elaborate a bit more on what this does, what values are accepted and maybe link to the appropriate documentation. This here maybe: https://docs.gitlab.com/ee/ci/environments/#scope-environments-with-specs ?

That will give the user a bit more context about that field.

Reply:
Sure. I added a link